README.md

```diff
@@ -25,132 +25,116 @@ Package.onUse(function (api) {
-imports.
+You can import styles from various locations, including other packages, your own app, and npm modules.
+
+- **Importing from another package**:
+
+```scss
+@use "meteor:{my-package:pretty-buttons}/buttons/styles" as buttons; // Assigns a namespace "buttons"
 
-Besides the usual way of importing files based on the relative path in the same
-package (or app), you can also import files from other packages or apps with the
-following syntax.
+.my-button {
+  @extend buttons.pretty-button; // Uses the imported class with its namespace
+}
+```
 
-Importing styles from a different package:
+or
 
 ```scss
-@import "{my-package:pretty-buttons}/buttons/_styles.scss";
+@use "meteor:{my-package:pretty-buttons}/buttons/styles" as *; // Imports everything into the global scope
 
 .my-button {
-  // use the styles imported from a package
-  @extend .pretty-button;
+  @extend .pretty-button; // No namespace required
 }
 ```
 
-Importing styles from the target app:
+- **Importing from your app**:
 
 ```scss
-@import "{}/client/styles/imports/colors.scss";
+@use "{}/client/styles/imports/colors.scss" as *;
 
 .my-nav {
-  // use a color from the app style pallete
-  background-color: @primary-branding-color;
+  background-color: @primary-branding-color; // Use a color from the app's style palette
 }
 ```
 
-This can also conveniently be used to import styles from npm modules for example:
+- **Importing from npm modules**:
+
 ```scss
-@import "{}/node_modules/module-name/stylesheet";
+@use "~module-name/stylesheet"; // Imports a module from node_modules
 ```
 
-Note that **Meteor 1.7** introduced a change so that files in `node_modules` aren't automatically compiled any more.
-This requires you to add a symlink inside the `imports` directory to the pacakge in order for compilation to work.
-E.g.
+If the target is a directory, it will search for an `index.scss`, `_index.scss`, `index.sass`, or `_index.sass` file inside that directory.
 
-```
-meteor npm install the-package
-cd imports
-ln -s ../node_modules/the-package .
-```
+### Global Include Path
 
-See the [meteor changelog](https://github.com/meteor/meteor/blob/devel/History.md) for more information.
+At the moment, there is no support for a global include path. If there is significant demand, this feature could be added in future updates.
 
-#### Global include path
+### Source Maps
 
-Sometimes a 3rd party module uses import paths that assume that the compiler is
-configured with specific `includePaths` option (e.g. Ionic, Bootstrap, etc.):
-```scss
-@import "ionicons-icons"; // This file is actually placed in another npm module!
-```
+Source maps are enabled by default, helping with debugging by mapping the compiled CSS back to the original Sass files.
+
+### Autoprefixer
+
+To use Autoprefixer, follow the official [PostCSS with Meteor](https://docs.meteor.com/packages/standard-minifier-css.html#standard-minifier-css) documentation.
```

Comment:
I'd add autoprefixer installation here as Meteor's article focused more on PostCSS

```sh
meteor npm install --save-dev autoprefixer postcss postcss-load-config
```

The rest can be learned from PostCSS Meteor docs article